apps/website/src/pages/institute/my-pixels.tsx

```diff
@@ -0,0 +1,91 @@
+import { type NextPage } from "next";
+import { useSession } from "next-auth/react";
+
+import { PIXEL_RENAME_LOCK_DURATION_TEXT } from "@/hooks/pixels";
+
+import Button from "@/components/content/Button";
+import Heading from "@/components/content/Heading";
+import Meta from "@/components/content/Meta";
+import Section from "@/components/content/Section";
+import { EditMyPixels } from "@/components/institute/EditMyPixels";
+import { EditPayPalPixels } from "@/components/institute/EditPayPalPixels";
+import { LoginWithTwitchButton } from "@/components/shared/LoginWithTwitchButton";
+import { MessageBox } from "@/components/shared/MessageBox";
+
+import IconArrowRight from "@/icons/IconArrowRight";
+
+import buildingHeroImage from "@/assets/institute/hero/building.png";
+
+const MyPixelsPage: NextPage = () => {
+  const session = useSession();
+
+  return (
+    <>
+      <Meta
+        title="My Pixels | Alveus Research & Recovery Institute"
+        description="View and edit your donated Pixels."
+        image={buildingHeroImage.src}
+      />
+
+      {/* Nav background */}
+      <div className="-mt-40 hidden h-40 bg-alveus-green-900 lg:block" />
+
+      <Section
+        dark
+        containerClassName="flex flex-wrap gap-4 justify-between items-end"
+      >
+        <div>
+          <Heading>Alveus Research & Recovery Institute My Pixels</Heading>
+          <p>
+            You will only be able to rename your Pixel once every{" "}
+            {PIXEL_RENAME_LOCK_DURATION_TEXT}. The last date to rename Pixels
+            will be announced on Alveus channels!
+          </p>
+        </div>
+        <Button href="/pixels" dark className="inline-flex items-center gap-2">
```

Comment:
```suggestion
        <Button href="/institute/pixels" dark className="inline-flex items-center gap-2">
```
Avoid sending the user through an extra redirect